#input
bool existing_settings
float64[] distortion
int64 camera_matrix_step
float64[] camera_matrix
---
#output
float64[] distortion
int64 camera_matrix_step
float64[] camera_matrix
int64 transform_matrix_step
float64[] transform_matrix
